base_station/gui/src/components/PDBFuse.vue

```diff
@@ -0,0 +1,305 @@
+        </tr>
+        <tr>
+          <td class="tableElement">Nichrome 2</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome2.current.color}">{{fuse_data.nichrome2.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome2.voltage.color}">{{fuse_data.nichrome2.voltage.val.toFixed(5)}}V</td>
+        </tr>
+        <tr>
+          <td class="tableElement">Nichrome 3</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome3.current.color}">{{fuse_data.nichrome3.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.nichrome3.voltage.color}">{{fuse_data.nichrome3.voltage.val.toFixed(5)}}V</td>
+        </tr>
+        <tr>
+          <td class="tableElement">HBridge 1</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.hbridge1.current.color}">{{fuse_data.hbridge1.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.hbridge1.voltage.color}">{{fuse_data.hbridge1.voltage.val.toFixed(5)}}V</td>
+        </tr>
+        <tr>
+          <td class="tableElement">HBridge 2</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.hbridge2.current.color}">{{fuse_data.hbridge2.current.val.toFixed(5)}}A</td>
+          <td class = "tableElement" v-bind:style="{color: fuse_data.hbridge2.voltage.color}">{{fuse_data.hbridge2.voltage.val.toFixed(5)}}V</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      pdb_data: {
+        three: {
+          temp: {val: 0.0, color: "grey"},
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        five: {
+          temp: {val: 0.0, color: "grey"},
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        twelve: {
+          temp: {val: 0.0, color: "grey"},
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        }
+      },
+
+      fuse_data: {
+        cytron1: {
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        cytron2: {
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        cytron3: {
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        cytron4: {
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        cytron5: {
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        cytron6: {
+          current: {val: 0.0, color: "grey"},
+          voltage: {val: 0.0, color: "grey"}
+        },
+        nichrome1: {
```

Comment:
same here, nichrome --> heater? also did rohan ask to relable them to A, B, C? or colors or something?